src/libs/actions/Report.js

```diff
@@ -170,12 +170,17 @@ function getSimplifiedReportObject(report) {
     const isLastMessageAttachment = /<img([^>]+)\/>/gi.test(lastActionMessage);
     const chatType = lodashGet(report, ['reportNameValuePairs', 'chatType'], '');
 
-    // We are removing any html tags from the message html since we cannot access the text version of any comments as
-    // the report only has the raw reportActionList and not the processed version returned by Report_GetHistory
-    // We convert the line-breaks in html to space ' ' before striping the tags
-    const lastMessageText = lastActionMessage
-        .replace(/((<br[^>]*>)+)/gi, ' ')
-        .replace(/(<([^>]+)>)/gi, '') || `[${translateLocal('common.deletedCommentMessage')}]`;
+    let lastMessageText;
```

Comment:
You could do this instead and save a couple of lines
```JS
    let lastMessageText = null;
    if (report.reportActionCount > 0) {
        // We are removing any html tags from the message html since we cannot access the text version of any comments as
        // the report only has the raw reportActionList and not the processed version returned by Report_GetHistory
        // We convert the line-breaks in html to space ' ' before striping the tags
        lastMessageText = lastActionMessage
            .replace(/((<br[^>]*>)+)/gi, ' ')
            .replace(/(<([^>]+)>)/gi, '') || `[${translateLocal('common.deletedCommentMessage')}]`;
    }
```

Reply:
Great catch. I have pushed the suggested changes.